.circleci/config.yml

```diff
@@ -22,8 +22,8 @@ jobs:
             - node_modules
           key: v1-dependencies-{{ checksum "package-lock.json" }}
         
-      - run: npm run check-style
+      - run: npm run format
       
-      - run: npm run test:coverage
+      - run: npm run test --coverage
```

Comment:
Shouldn't this be `npm run test -- --coverage`?

Reply:
Oops, I confused it with yarn